lib/resources/BulkSendJob.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (C) 2015 hellosign.com
```

Comment:
Should we update the Copyright year in the notes to 2020?

Reply:
Good call